substrate/frame/assets/src/lib.rs

```diff
@@ -1798,6 +1848,92 @@ pub mod pallet {
 			)?;
 			Ok(())
 		}
+
+		/// Mint a distribution of assets of a particular class.
```

Comment:
Something about this can go into the main top docs and then in https://docs.rs/pallet-assets/latest/pallet_assets/?